CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). (Patch version X.Y.0 is implied if not specified.)
 
 ## [Unreleased](https://github.com/USGS/time-series-services/compare/time-series-services/compare/0.6.0...master)
-
+### Changed
+-   IOW-506 Unapproved data older than 1095 is not returned
```

Comment:
Thanks for updating the change log.  Since the change log is meant for wider public the Jria number (IOW-506)
should not be in there:
Unapproved time series observations older than 1095 days are not returned.

Reply:
Fixed.